canonical-web-and-design/jaas-dashboard
# Update run script to 3.4.3

## Done

Update run script to 3.4.3

## QA

- Pull code
- `./run clean && ./run`
- Open http://0.0.0.0:8036/
- It should work as expected.
